Fix:

error: implicitly declaring library function 'strcmp' with type 'int (const char *, const char *)' [-Werror,-Wimplicit-function-declaration]

error: implicitly declaring library function 'strlen' with type 'unsigned long (const char *)' [-Werror,-Wimplicit-function-declaration]

error: implicitly declaring library function 'strerror' with type 'char *(int)' [-Werror,-Wimplicit-function-declaration]
--- timelimit.c.orig	1998-10-28 14:30:06.000000000 -0600
+++ timelimit.c	2020-12-07 03:12:52.000000000 -0600
@@ -24,6 +24,7 @@
 
 #include <stdio.h>
 #include <signal.h>
+#include <string.h>
 #include <time.h>
 #include <sys/types.h>
 #include <sys/wait.h>
